tools/getting-started/src/error.tsx

```diff
@@ -0,0 +1,16 @@
+import { Text } from "ink";
+import React from "react"
+
+interface Props {
```

Comment:
we use 2 space padding everywhere in the project. make sure the new files are consistent with this

Reply:
I'm a bit confused with your comment as all my files use 2 spaces...